Power Up Proto/src/org/usfirst/frc/team5493/robot/Robot.java

```diff
@@ -35,10 +43,11 @@
     
     public void robotInit() {
     	
-    	climber = new Climber();
+    	climber = new OneClimbyBoi();
     	cubeControls = new CubeControls();
+    	throwDaggersInBensEyes = new ThrowDaggersInBensEyes();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)